app/preprints/detail/template.hbs

```diff
@@ -109,6 +109,13 @@
             @latestAction={{this.model.latestAction}}
         />
     {{/if}}
+    {{#if this.showOSFBanner}}
+        <Preprints::-Components::PreprintStatusBanner
+            @submission={{this.model.preprint}}
+            @provider={{this.model.provider}}
+            @isOSFBanner=true
```

Comment:
Minor nit: I think this should be `{{true}}`, though it seems like it's working fine as is?